### ISSUE TITLE
fbjs missing

### ISSUE THREAD

**Issue by DevGorilla** [iddan/react-spreadsheet#24]
**Error:**
<img width="1035" alt="Screen Shot 2019-03-08 at 8 53 30 PM" src="https://user-images.githubusercontent.com/6393813/54066575-0514a280-41e7-11e9-888a-79e739524acb.png">

**usage:**
```
import React from 'react';
import Spreadsheet from "react-spreadsheet";

class Expenses extends React.Component {
  constructor (props, context) {
    super(props, context);
  
  }
  render () {
    const data = [
  [{ value: "Vanilla" }, { value: "Chocolate" }],
  [{ value: "Strawberry" }, { value: "Cookies" }]
];
    return (
        <Spreadsheet data={data} />
    )
  }
}

export default Expenses;
```

**fix:** npm install fbjs --save

**Comment by iddan** [comment]
Thank you for highlighting that. Will try to triage as soon as possible. Accepting PRs.

**Comment by cdrani** [comment]
Is the fix as simple as shown above?

**Comment by DevGorilla** [comment]
Yes. I am unable to greate a pull request from my fork. But the solution to have fbjs included in the package.json. https://github.com/DevGorilla/React-Spreadsheet-Component

**Comment by iddan** [comment]
Fixed in v0.4.27
